lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -3443,6 +3443,9 @@ bool LLVMToSPIRVBase::translate() {
   if (isEmptyLLVMModule(M))
     BM->addCapability(CapabilityLinkage);
 
+  // Transform SPV-IR builtin calls to builtin variables.
+  transWorkItemBuiltinsToVariables(M);
```

Comment:
This function is also called from OCLToSPIRV pass. I guess it is called twice because built-in calls instead variables are expected for SPIR-V friendly IR as well. So, maybe we make this transformation common and for example do only one call to this function but from some regularization pass which runs for all source languages?